pkgs/applications/blockchains/groestlcoin/default.nix

```diff
@@ -112,7 +110,16 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
-  meta = with lib; {
+  __darwinAllowLocalNetworking = true;
+
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+  versionCheckProgram = "${placeholder "out"}/bin/groestlcoin-cli";
+  versionCheckProgramArg = "--version";
+  doInstallCheck = true;
```

Comment:
I think there is a newer hook to do this. Can you check other packages to convert this?